### PR TITLE
Add thorough explanations to the visualizations

### PR DESCRIPTION
**Pull request by JonathanChavezTamales**
### Summary

In this PR I add more explanations to the visualizations and more context to the sections because some days ago we just had the raw visualizations and just one liners that weren't that much explanatory.

Feel free to point some changes, I tried to do it the best I could but maybe there are many things that still can improve.

### CODE REVIEW

File: src/views/mixed-content/government/index.js [hunk 1]
```diff
@@ -7,11 +8,26 @@ import {Typography} from '@material-ui/core';
...
+      <Typography variant="h2" gutterBottom={true}>Government</Typography>
+      <Typography variant="overline" gutterBottom={true}>
+        Summary of mixed content present in <b>government websites</b>.
       </Typography>
-      <TopCountriesWithMoreWebsitesWithMixedContent />
+      <Typography variant="h6">
+        Why analyze government websites?
+      </Typography>
+      <Typography paragraph={true}>
+        Governmental webistes are unfortunately the ones that contain the most percentage of <b>mixed content</b> resources,
```
[the remark this answers is not in the document]
Comment (JonathanChavezTamales) [reply src/views/mixed-content/government/index.js:19]: Done!

File: src/views/mixed-content/worldwide/mixed-content-percentage-histogram/index.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import CircularProgress from '@material-ui/core/CircularProgress';
 import MuiAlert from '@material-ui/lab/Alert';
 import {ResponsivePie} from '@nivo/pie';
 import {Snackbar} from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
```
[the remark this answers is not in the document]
Comment (JonathanChavezTamales) [reply src/views/mixed-content/worldwide/mixed-content-percentage-histogram/index.js:10]: Done!

File: src/views/mixed-content/government/index.js [hunk 3]
```diff
@@ -7,11 +8,26 @@ import {Typography} from '@material-ui/core';
...
+        this makes them a great attack target because they are popular and there's a great flux of sensitive information in them to be sniffed.
```
[the remark this answers is not in the document]
Comment (JonathanChavezTamales) [reply src/views/mixed-content/government/index.js:20]: Done! Thanks for letting me know.

File: server/__tests__/api.test.js [hunk 4]
```diff
@@ -197,6 +197,35 @@ describe('mixed-content', () => {
     expect(res.body.result).toBeInstanceOf(Array);
     expect(res.body.suggestedVisualizations).toContain('Line chart');
   });
+
+  it('Check /mixed-content/mixed-content-by-type with type = all', async () => {
+    const res = await request(app).get(
+      '/api/mixed-content/mixed-content-by-type?type=all',
+    );
+    expect(res.body.description).toBe('Return mixed content request by types');
+    expect(res.body.result).toMatchObject([
```
[the remark this answers is not in the document]
Comment (JonathanChavezTamales) [reply server/__tests__/api.test.js:206]: Changed to toBeInstanceOf(Array), I handled it that way in the other tests.

File: src/views/mixed-content/government/top-countries-with-more-websites-with-mixed-content/elements.js [hunk 5]
```diff
@@ -5,4 +5,10 @@ const CustomCard = styled(Card)`
   margin: 20px 0px;
 `;
 
-export {CustomCard};
+const SkeletonContainer = styled.div`
+  width: 100%;
+  padding: 20px;
+`;
+
+export {ChartContainer, CustomCard, SkeletonContainer};
```
[the remark this answers is not in the document]
Comment (JonathanChavezTamales) [reply src/views/mixed-content/government/top-countries-with-more-websites-with-mixed-content/elements.js]: Done, thanks for pointing out

File: src/views/mixed-content/government/top-countries-with-more-websites-with-mixed-content/index.js [hunk 6]
```diff
@@ -8,7 +8,16 @@ import {CustomCard} from './elements';
 import MuiAlert from '@material-ui/lab/Alert';
 import {ResponsiveChoropleth} from '@nivo/geo';
 import {Snackbar} from '@material-ui/core';
+import {ChartContainer, CustomCard, SkeletonContainer} from './elements';
```
[the remark this answers is not in the document]
Comment (JonathanChavezTamales) [reply src/views/mixed-content/government/top-countries-with-more-websites-with-mixed-content/index.js]: done

File: src/views/mixed-content/government/index.js [hunk 7]
```diff
@@ -7,11 +8,28 @@ import {Typography} from '@material-ui/core';
...
+      <Typography variant="h2" gutterBottom={true}>
+        Government
       </Typography>
-      <TopCountriesWithMoreWebsitesWithMixedContent />
+      <Typography variant="overline" gutterBottom={true}>
+        Summary of mixed content present in <b>government websites</b>.
+      </Typography>
+      <Typography variant="h6">Why analyze government websites?</Typography>
+      <Typography paragraph={true}>
+        Governmental websites are unfortunately the ones that contain the most
```
Comment (carlosjoan91) [inline src/views/mixed-content/government/index.js:19]: Nit: I would probably suggest not making a conclusion of whether or not they are good attack targets, and instead suggest  something like "Government websites contain a large amount of mixed content, which can be particularly risky due to them also handling sensitive infromation."

Comment (JonathanChavezTamales) [reply src/views/mixed-content/government/index.js:19]: Thanks for letting me know! I'll convert this into an issue.

**Pull request merged**